src/Pages/ContentListTable/ContentListTable.tsx

```diff
@@ -453,6 +465,27 @@ const ContentListTable = () => {
                           {name}
                           <br />
                           <UrlWithExternalIcon href={url} />
+                          {features?.snapshots?.accessible && 
```

Comment:
So this is a funny one, there are times when doing things like this will return the text value of "unknown" "undefined" or "false" into the final html.

So in this case using a ternary operator with an empty fragment would be ideal. 
BUT we do have a custom wrapper component that we can use instead that handles this, simply called
` <Hide hide={boolean}>`

You should likely use that here, and below where the "last_snapshot" is being checked.

Reply:
ah gotcha! updated to use Hide